dashboard/.streamlit/config.toml

```diff
@@ -4,7 +4,7 @@ gatherUsageStats = false
 [theme]
 base = "light"
 primaryColor = "#3c9aaa"
```

Comment:
Would be nice to have a comment above or next to this saying what the colour is (in words)

Reply:
That's a nice addition, I'll add that now